### ISSUE TITLE
Checking Submersion task does not reduce submersion cost by 20%

### ISSUE THREAD

**Issue by Lochabar** [chummer5a/chummer5a#2284]
### Environment
Chummer Version: 5.196.39
Environment: Windows 7 Home Premium (64-bit)
Runtime: (probably .NET 4.7.1)

### Expected behaviour
When Submersion task is checked the karma cost of submersion is reduced by 20%

### Actual behaviour
Only reduced by 1 karma which is no more than 10%

Without submersion task checked karma cost is 13
![without task](https://user-images.githubusercontent.com/17562417/34756061-0b67898c-f5c0-11e7-9980-85bd9c5de8a9.png)

When submersion task is checked karma cost is 12
![with task](https://user-images.githubusercontent.com/17562417/34756077-2b92f1e2-f5c0-11e7-9694-0b215790cdfa.png)

Data Trails p164 shows the reduction should be 20% (rounded up)
![task](https://user-images.githubusercontent.com/17562417/34756118-84ada628-f5c0-11e7-8179-3bebc20e68aa.png)
which should be 2.6 rounded up to 3 for a net karma cost of 10 (not 12)

### Steps to reproduce the behaviour
Prepare a technomancer for first level submersion.

**Comment by Lochabar** [comment]
Still not 20% rounded up - 20% of 13 is 2.6 rounding up to 3. Therefore net cost should be 10 karma and not 11 as performed in 5.197.3
![with task 2](https://user-images.githubusercontent.com/17562417/35140866-b3ad5e96-fcf0-11e7-97eb-34d57c32a546.png)

**Comment by chummer5a** [comment]
It rounds the Karma cost upwards, not the discount. I'd need to see errata indicating otherwise.

**Comment by Lochabar** [comment]
I checked the Questions for the SRM FAQ Committee and it seems you have the correct calculation. The question referred to initiation discounts rather that submersion but its only logical the same ruling would apply. From the website:

Initiatory discounts, how are they rounded? The default is to round up, but which part do we round? The types of discounts are listed as: Base Price - 10%, so my gut is to round the final price back up. But I could see some people rounding the discount amount up instead (resulting in a larger discount).

Example: First couple of initiations, assuming we've joined the Ash Union and manage to perform an Ordeal every time (for some extra fun).

Initiation 1: Base 13 karma. w/ 20% discount -> 13 - 20% (2.6) = 10.4 (rounded up) so 11
